app/containers/message/Markdown.js

```diff
@@ -38,14 +38,15 @@ export default class Markdown extends React.Component {
 		let m = formatText(msg);
 		m = emojify(m, { output: 'unicode' });
 		m = m.replace(/^\[([^\]]*)\]\(([^)]*)\)/, '').trim();
+		let p_edited = edited;
```

Comment:
please change variable name and use camelCase

Reply:
done